### ISSUE TITLE
Issue with a11ychecker CKEditor plugin results in zero width space characters being inserted into HTML content

### ISSUE THREAD

**Issue by no3rdw** [MasaCMS/MasaCMS#229]
**Describe the bug**
CK Editor inserts seven \u200b unicode characters (zero width spaces) when inserting an image into an HTML area via the inline editor toolbar.  When the page is saved, these characters are converted to question marks.  This does not occur when inserting images via 'Full Edit' mode.  

**To Reproduce**
- Open any page in Inline Edit mode,
- Double click the Body or other HTML area to begin editing
- Click the toolbar button to insert an image
- Choose an image from the server or type in an image URL (sample.png)
- View the HTML source via CK Editor toolbar to see seven \u200b unicode characters inserted after the image tag
- Save the changes to the page to see these get converted to question marks

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome and Edge

**Environment (please complete the following information):**
 - OS: Windows Server
 - CFML Engine: Adobe ColdFusion 2018
 - Database MSSQL
 - Masa Versions: Observed in 7.3.11 and 7.4.3

**Comment by jeroenweareorangeeu** [comment]
So I also came across this bug while looking into another ticket. I investigated further and this is what I think is happening.

So 'sometimes' the A11yChecker plugin throws a JS error saying:

_Uncaught ReferenceError: jQuery is not defined_
![a11y_js_error](https://github.com/MasaCMS/MasaCMS/assets/83004496/edbca531-8e9d-49c1-99ef-e64e717740cf)

What I found is that the A11yyChecker depends on jQuery to be available (as stated in the a11ychecker docs btw).
In Masa this means that jQuery has to be used in the Masa CMS Theme that you are using for your site.
That means that this error does not occurr when using the 'default' Masa Bootstrap **4** theme, 
but it DOES occur when using the Masa Bootstrap **5** theme which doesnt use jQuery anymore!

A solution could be to remove this plugin from CKEditorin Masa CMS. Or somehow jQuery is provided for this plugin.
